### PR TITLE
fix(drawer): hierarchy layout

### PR DESCRIPTION
**Pull request by kikostadinov**
Fixes: https://github.com/telerik/kendo-themes/issues/4010

### CODE REVIEW

**Review by TeyaVes — COMMENTED** [review]
I don't see any changes in the variables file in the classic theme, as well as in the layouts and variables files for fluent. Aren't we adding the fix there as well?

File: packages/bootstrap/scss/drawer/_layout.scss [hunk 1]
```diff
@@ -15,6 +15,12 @@
                     padding-left: $drawer-item-padding-x;
                 }
             }
+
+            // Fallback styles when drawer item is missing k-spacer element
+            .k-i-arrow-chevron-up,
+            .k-i-arrow-chevron-down {
```
Comment (Juveniel) [inline packages/bootstrap/scss/drawer/_layout.scss:21]: Targeting specific icon classes is not a good idea and flaky. Find another way.

File: packages/bootstrap/scss/drawer/_variables.scss [hunk 2]
```diff
@@ -8,7 +8,7 @@ $drawer-font-size: $font-size !default;
 $drawer-line-height: $line-height !default;
 $drawer-content-padding-x: $padding-x !default;
 $drawer-content-padding-y: $padding-y !default;
-$drawer-item-level-padding-x: k-map-get( $spacing, 4 ) !default;
+$drawer-item-level-padding-x: $drawer-content-padding-x !default;
```
Comment (Juveniel) [inline packages/bootstrap/scss/drawer/_variables.scss:11]: Do we want to modify the hierarchy items padding when changing the padding of the drawer content ? I think no, thus, better use `$padding-x` here.

**Review by Juveniel — CHANGES_REQUESTED** [review]
One additional thing - it seems from the changed drawer visual tests, that the padding of the first level is bigger than the rest. Also, the change is only visible in the classic, default and fluent themes, but not in bootstrap and material. Can you have a look at this ?

File: packages/bootstrap/scss/drawer/_layout.scss (hunk 1 above)
Comment (joneff) [reply packages/bootstrap/scss/drawer/_layout.scss:21]: if this is "unification" the class must be present!

File: packages/bootstrap/scss/drawer/_layout.scss [hunk 3]
```diff
@@ -15,6 +15,12 @@
...
+                margin-inline-start: auto;
+            }
         }
 
         .k-rtl & .k-drawer-items,
```
Comment (joneff) [inline packages/bootstrap/scss/drawer/_layout.scss:26]: use logical padding

**Review by joneff — CHANGES_REQUESTED** [review]

**Review by joneff — APPROVED** [review]

**Comment by joneff** [comment]
@TeyaVes can you add the breaking change note in your commit. Apart from that minor thingie, lgtm

**Review by TeyaVes — APPROVED** [review]

**Review by joneff — APPROVED** [review]

**Pull request merged**